test/src/test/java/org/corfudb/runtime/object/transactions/StreamTest.java

```diff
@@ -45,6 +51,39 @@
     private final int MAX_PERCENT = 100;
 
 
+    @Test(expected = AppendException.class)
```

Comment:
Don't use the JUnit expected= annotation, please wrap the statement that should throw the exception in an assertion and assert that statement throws the correct exception.